### PR TITLE
#123 data source password encryption

### PR DESCRIPTION
**Pull request by alexisrolland**

### CODE REVIEW

**Comment by alexisrolland** [comment]
Unit tests in progress

File: db/init/05-data-source.sql [hunk 1]
```diff
@@ -3,6 +3,20 @@
 
 
 
+/*Create function to encrypt password column*/
+CREATE OR REPLACE FUNCTION base.encrypt_password()
+RETURNS TRIGGER AS $$
+BEGIN
+    NEW.password = PGP_SYM_ENCRYPT(NEW.password,'AES_KEY');
```
Comment (ThomasGassmann) [inline db/init/05-data-source.sql]: Add a space after the comma: `(NEW.password, 'AES_KEY')`
Also: where's the `AES_KEY` coming from?

Comment (ThomasGassmann) [reply db/init/05-data-source.sql]: I think we should have different keys for different scenarios though. Additionally the keys we already have are used for asymmetric encryption. This is symmetric encryption.

Comment (alexisrolland) [reply db/init/05-data-source.sql]: To keep the implementation simple, I have done the following:
- create a new schema and table configuration.parameter
- generate secret key during database setup and store it in configuration.parameter
- update trigger and view to encrypt data using the key

**Pull request merged**